k8s/helm/charts/accounts/templates/deployment.yaml

```diff
@@ -41,6 +41,11 @@ spec:
           env:
 
             # ConfigMap
+            - name: CHARGE_BASE_URL
```

Comment:
@NikolayRn please delete this ConfigMap configuration, non - hardcoded ConfigMaps inserted automatically. Just FYI - https://www.notion.so/Kubernetes-Update-or-add-Helm-chart-environment-variables-0691d192cd0a4fb7afce54fbcbbe0474.